<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashboardWindow</class>
 <widget class="QMainWindow" name="DashboardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema Escolar - Dashboard Mockup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f5f6fa;
}
#sidebar {
    background-color: #2c3e50;
    border: none;
}
#sidebarTitle {
    color: white;
    font-size: 24px;
    font-weight: bold;
    font-family: 'Segoe UI';
}
QPushButton {
    background-color: transparent;
    color: #ecf0f1;
    text-align: left;
    padding: 15px;
    border-radius: 5px;
    font-size: 14px;
    font-family: 'Segoe UI';
}
QPushButton:hover {
    background-color: #34495e;
}
#userLabel {
    color: #95a5a6;
    font-size: 12px;
}
#contentArea {
    background-color: #f5f6fa;
}
#headerTitle {
    font-size: 28px;
    color: #2c3e50;
    font-weight: bold;
    font-family: 'Segoe UI';
}
QTableWidget {
    background-color: white;
    border-radius: 10px;
    border: 1px solid #dcdde1;
    padding: 10px;
    gridline-color: #f5f6fa;
}
QHeaderView::section {
    background-color: white;
    padding: 10px;
    border: none;
    border-bottom: 2px solid #f5f6fa;
    font-weight: bold;
    color: #7f8c8d;
}
QTableWidget::item {
    padding: 10px;
    border-bottom: 1px solid #f5f6fa;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="sidebar">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>40</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="sidebarTitle">
         <property name="text">
          <string>SCHOOL
SYSTEM</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnEstudiantes">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>üë§ Gestionar Estudiantes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCursos">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>üìö Gestionar Cursos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnMatricula">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>üìã Gestionar Matr√≠culas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAsistencia">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>üìÖ Tomar Asistencia</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNotas">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>üìù Registro de Notas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnReportes">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>üìä Reportes</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="userLabel">
         <property name="text">
          <string>GRUPO 1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="contentArea" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>30</number>
       </property>
       <property name="leftMargin">
        <number>40</number>
       </property>
       <property name="topMargin">
        <number>40</number>
       </property>
       <property name="rightMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>40</number>
       </property>
       <item>
        <widget class="QLabel" name="headerTitle">
         <property name="text">
          <string>Dashboard General</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="kpiContainer" native="true">
         <layout class="QHBoxLayout" name="kpiLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="cardEstudiantes">
            <property name="styleSheet">
             <string notr="true">QFrame {
    background-color: white;
    border-radius: 10px;
    border-left: 5px solid #3498db;
}
QLabel#lblValEstudiantes {
    font-size: 24px;
    font-weight: bold;
    color: #2c3e50;
}
QLabel#lblTitEstudiantes {
    color: #7f8c8d;
    font-size: 14px;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="vbox1">
             <item>
              <widget class="QLabel" name="lblValEstudiantes">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblTitEstudiantes">
               <property name="text">
                <string>Total Estudiantes</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="cardCursos">
            <property name="styleSheet">
             <string notr="true">QFrame {
    background-color: white;
    border-radius: 10px;
    border-left: 5px solid #2ecc71;
}
QLabel#lblValCursos {
    font-size: 24px;
    font-weight: bold;
    color: #2c3e50;
}
QLabel#lblTitCursos {
    color: #7f8c8d;
    font-size: 14px;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="vbox2">
             <item>
              <widget class="QLabel" name="lblValCursos">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblTitCursos">
               <property name="text">
                <string>Total Cursos</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="cardPromedio">
            <property name="styleSheet">
             <string notr="true">QFrame {
    background-color: white;
    border-radius: 10px;
    border-left: 5px solid #f1c40f;
}
QLabel#lblValPromedio {
    font-size: 24px;
    font-weight: bold;
    color: #2c3e50;
}
QLabel#lblTitPromedio {
    color: #7f8c8d;
    font-size: 14px;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="vbox3">
             <item>
              <widget class="QLabel" name="lblValPromedio">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblTitPromedio">
               <property name="text">
                <string>Promedio General</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="cardRiesgo">
            <property name="styleSheet">
             <string notr="true">QFrame {
    background-color: white;
    border-radius: 10px;
    border-left: 5px solid #e74c3c;
}
QLabel#lblValRiesgo {
    font-size: 24px;
    font-weight: bold;
    color: #c0392b;
}
QLabel#lblTitRiesgo {
    color: #7f8c8d;
    font-size: 14px;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="vbox4">
             <item>
              <widget class="QLabel" name="lblValRiesgo">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblTitRiesgo">
               <property name="text">
                <string>Alumnos en Riesgo</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>√öltimas Calificaciones</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="columnCount">
          <number>8</number>
         </property>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nombre</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Curso</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nota 1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nota 2</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nota 3</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Promedio</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Estado</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
